iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/TwoWindingsTransformerImpl.java

```diff
@@ -177,6 +178,11 @@ public PhaseTapChangerImpl getPhaseTapChanger() {
         return tapChangers;
     }
 
+    @Override
+    public Set<TapChanger> getAllTapChangersOnLeg() {
+        return getAllTapChangers();
+    }
```

Comment:
Don't think this method is necessary, see other remarks

Reply:
Deleted